quranref/cli.py

```diff
@@ -0,0 +1,88 @@
+import typer
+from time import sleep
+from .graph_models import get_gdb
+from .graph_models.quran_graph import QuranGraph
+from .graph_models.quran_graph import Surah
+from .lib.surah_info import surah_info
+from .graph_models.quran_graph import Aya, AyaText
+
+cmd_app = typer.Typer()
+"""
+def iteration():
+    for i in range(1000):
+        yield i
+"""
+
+@cmd_app.command()
+def populate():
+    """Create database tables and graph."""
+    print("Creating tables and graph")
+    gdb = get_gdb()
+    db_objects = [QuranGraph]
+    gdb.create_all(db_objects)
+
+@cmd_app.command()
+def import_surahs():
+    surah_number = 1
+    for s_info in surah_info:
+        s = Surah(
+            _key=str(surah_number),
+            surah_number=surah_number,
+            arabic_name=s_info['arabic_name'],
+            english_name=s_info['english_name'],
+            translated_name=s_info['translated_name'],
+            nuzool_location=s_info['nuzool_location'],
+            nuzool_order=s_info['nuzool_order'],
+            rukus=s_info['rukus'],
+            total_ayas=s_info['total_ayas']
+        )
+
+        gdb = get_gdb()
```

Comment:
You're creating database connection inside the loop so it will be created for each iteration of the loop which is bad for performance. Database connection could have been created before the loop and closed after the loop. Within the loop you could have just done `gdb.add(s)`